docs/en/table_design/table_types/Comparison_of_abilities_of_various_tables.md

```diff
@@ -0,0 +1,43 @@
+</table>
+
+## Compatibility with other features
+
+
+<table>
+<thead>
+<tr><th colspan="2"></th><th><strong>Primary Key table</strong></th><th><strong>Duplicate Key table</strong></th><th><strong>Aggregate table</strong></th><th><strong>Unique Key table</strong></th></tr>
+</thead>
+<tbody><tr><td rowspan="2"><strong>Bitmap index/bloom filter index</strong></td><td><strong>Build indexes on key columns</strong></td><td colspan="4">Supported</td></tr><tr><td><strong>Build indexes on value columns</strong></td><td>Supported</td><td>Supported</td><td>Not supported</td><td>Not supported</td></tr><tr><td rowspan="2"><strong>Partitioning/bucketing</strong></td><td><strong>Expression partitioning/list partitioning</strong></td><td colspan="4">Supported</td></tr><tr><td><strong>Random bucketin</strong></td><td>Not supported</td><td>Supported from v3.1 onwards.</td><td>Not supported</td><td>Not supported</td></tr><tr><td  rowspan="2"><strong>Materialized views</strong></td><td ><strong>Asynchronous materialized views</strong></td><td colspan="4">Supported</td></tr><tr><td><strong>Synchronous materialized views</strong></td><td>Not supported</td><td>Supported</td><td>Supported</td><td>Supported</td></tr><tr><td rowspan="2"><strong>Other features</strong></td><td><strong>CTAS</strong></td><td>Supported</td><td>Supported</td><td>Not supported </td><td>Not supported </td></tr><tr><td><strong>backup &amp; restore</strong></td><td>Supported from v2.5 onwards.</td><td colspan="3">Supported</td></tr></tbody>
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/a1d625c7-1bf9-4603-bb9d-3cfe41339546)
remove the periods (.)